systemtest/mauveLoadTest/playlist.xml

```diff
@@ -30,9 +30,8 @@
 		</impls>
 		<platformRequirements>^os.osx</platformRequirements>
 	</test>
-	
 	<test>
-		<testCaseName>MauveSingleThreadLoadTest_HS</testCaseName>
+		<testCaseName>MauveSingleThreadLoadTest_OpenJ9</testCaseName>
```

Comment:
I think that MauveSingleThreadLoadTest_OpenJ9 and MauveSingleThreadLoadTest_HS can be collapsed into 1 target again, with no <impls> in the target, as for 11+ the tests run on both impls on all platforms.  The only reason to split them would be in the event we have to exclude on a platform for a particular impl (as is the case for osx jdk8 j9).  Whenever we get a chance, we should re-unite the targets in playlists (that get split temporarily to accommodate tmp excludes and such.

Reply:
> Whenever we get a chance, we should re-unite the targets in playlists (that get split temporarily to accommodate tmp excludes and such.

Agreed. 

`MauveSingleThreadLoadTest_OpenJ9` is jdk11+ only, whereas `MauveSingleThreadLoadTest_HS` is not.

I will collapse `MauveSingleThreadLoadTest_HS` and `MauveSingleThreadLoadTest_OpenJ9` (and also the other two mauve test pairs) - once we have it working on osx jdk9 on openj9.